src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParserStateObject.cs

```diff
@@ -244,6 +247,41 @@ sealed internal class TdsParserStateObject
 #pragma warning restore 0649
 #endif
 
+#if MANAGED_SNI
+
+        internal SspiClientContextStatus sspiClientContextStatus = new SspiClientContextStatus();
+
+        internal class SspiClientContextStatus
+        {
+            private ContextFlagsPal _contextFlags = ContextFlagsPal.None;
+
+            public SafeFreeCredentials CredentialsHandle
+            {
+                get;
+                set;
+            }
+
+            public SafeDeleteContext SecurityContext
+            {
+                get;
+                set;
+            }
+
+            public ContextFlagsPal ContextFlags
```

Comment:
You could even get rid of the explicit ctor if you wanted, e.g.
```C#
public ContextFlagsPal ContextFlags { get; set; } = ContextFlagsPal.None;
```
That said, None is the default value (0), so you don't  need to set it at all:
```C#
public ContextFlagsPal ContextFlags { get; set; }
```